qutebrowser/utils/utils.py

```diff
@@ -740,3 +741,29 @@ def ceil_log(number, base):
         result += 1
         accum *= base
     return result
+
+
+def list_tabs():
+    """
+    A method that returns a list of open buffers
+    """
+    tabs = []
+
+    for win_id in objreg.window_registry:
+        tabbed_browser = objreg.get('tabbed-browser', scope='window', window=win_id)
+
+        for idx in range(tabbed_browser.widget.count()):
+            tab = tabbed_browser.widget.widget(idx)
+            tabs.append((tab.url().toDisplayString(),
+                tabbed_browser.widget.page_title(idx),
+                "{}/{}".format(win_id, idx + 1)))
+
+    return tabs
+
+
+def tuple_for_url_in_tab(cur_url):
```

Comment:
Unless I'm missing something, you only call this in one place, and that place only uses the `title`, so why not just return that?

Reply:
It only used the index (`0/1`). It now just returns it.